src/components/slider-prompt/slider-prompt.jsx

```diff
@@ -0,0 +1,130 @@
+import classNames from 'classnames';
+import {defineMessages, FormattedMessage, intlShape, injectIntl} from 'react-intl';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Box from '../box/box.jsx';
+import Modal from '../../containers/modal.jsx';
+
+import styles from './slider-prompt.css';
+
+
+const messages = defineMessages({
+    minValue: {
+        defaultMessage: 'Minimum value',
+        description: 'Label of slider modal',
+        id: 'gui.sliderModal.min'
+    },
+    maxValue: {
+        defaultMessage: 'Maximum value',
+        description: 'Label of slider modal',
+        id: 'gui.sliderModal.max'
+    },
+    title: {
+        defaultMessage: 'Change slider range',
+        description: 'Title of slider modal',
+        id: 'gui.sliderModal.title'
+    },
+    decimal: {
+        defaultMessage: 'Handle as decimal value',
+        description: 'Label of a checkbox which is checked when it should handle decimal values.',
+        id: 'gui.sliderModal.decimal'
+    }
+});
+
+const SliderPromptComponent = props => (
+    <Modal
+        className={styles.modalContent}
+        contentLabel={props.intl.formatMessage(messages.title)}
+        id="sliderPrompt"
+        onRequestClose={props.onCancel}
+    >
+        <Box className={styles.body}>
+            <Box className={styles.label}>
+                {props.intl.formatMessage(messages.minValue)}
+            </Box>
+            <Box>
+                <input
+                    autoFocus
+                    className={styles.minInput}
+                    defaultValue={props.defaultMinValue}
+                    name={props.intl.formatMessage(messages.minValue)}
+                    step="0.01"
+                    type="number"
```

Comment:
We generally avoid `type="number"` for inputs since the default browser behavior is not good in a very interactive application (hijacks scrolling, adds extra UI we do not want, etc.)